tgui/packages/tgui/interfaces/CameraConsole.js

```diff
@@ -100,36 +100,45 @@ export const CameraConsoleContent = (props, context) => {
   const { activeCamera } = data;
   const cameras = selectCameras(data.cameras, searchText);
   return (
-    <Fragment>
-      <Input
-        autoFocus
-        fluid
-        mb={1}
-        placeholder="Search for a camera"
-        onInput={(e, value) => setSearchText(value)} />
-      <Section>
-        {cameras.map(camera => (
+    <Flex
+      direction={"column"}
+      height="100%">
+      <Flex.Item>
+        <Input
+          autoFocus
+          fluid
+          mb={1}
+          placeholder="Search for a camera"
+          onInput={(e, value) => setSearchText(value)} />
+      </Flex.Item>
+      <Flex.Item
+        height="90%">
```

Comment:
Should be grow={1}

Reply:
That causes the flex area to expand to the length of the section. I want the flex itself to instead be scroll-able, so that the search bar above it is always visible.

Reply:
> That causes the flex area to expand to the length of the section.

How